### ISSUE TITLE
Xak 2 (1990)(Microcabin) - Visual glitch at the opening

### ISSUE THREAD

**Issue by saebaryo** [MiSTer-devel/PC88_MiSTer#19]
Single SDRAM v3.00 128MB
PC88_unstable_20241205_17a328

Mode :  N88V2
Speed : 8Mhz

![image](https://github.com/user-attachments/assets/840adcee-ae13-4125-81ff-45b741669ba3)

![image](https://github.com/user-attachments/assets/f162ae06-d979-46e0-b8cc-4dd228a769cc)

![image](https://github.com/user-attachments/assets/912a1078-e6a8-4b1f-bbd0-9d1ce1636203)

![image](https://github.com/user-attachments/assets/981a254a-0f65-44f2-8334-e9e052ec2f76)

![image](https://github.com/user-attachments/assets/611fbf11-5197-4042-b971-a11c2a9d2940)

![image](https://github.com/user-attachments/assets/4a26a692-8cf1-4c7a-8768-9cead6c00d35)

![image](https://github.com/user-attachments/assets/992ae39d-5443-4270-adc4-ad1c69c0db6b)

**Comment by wwark** [comment]
It is fixed thanks Yosinda !

**Comment by saebaryo** [comment]
Single SDRAM v3.00 128MB
PC88_unstable_20241214_1372d9

Mode : N88V2
Speed : 8Mhz

![image](https://github.com/user-attachments/assets/1f1054fb-ba06-46a1-bcef-42dacc525891)

My MiSTerFPGA broke down, so I'm late to check getting a new MiSTer.

Thank you for the fix.
However, I used a new core (PC88_unstable_20241214_1372d9), but only the cursor flashed on the black screen and did not proceed any further. (The cursor is in the upper left corner of the above screenshot.)

Of course, it boots normally in the previous core (PC88_unstable_20241205_17a328) and enters the opening sequence.

Anyway, thank you for the core improvement.

**Comment by saebaryo** [comment]
Single SDRAM v3.00 128MB
**PC88_unstable_20241214_1372d9
PC88_unstable_20241221_11b8df**

Mode : N88V2
Speed : 8Mhz

Earlier in reply, I forgot to set up the disk on FDD1. I confirmed that it is fixed on two cores: PC88_unstable_20241214 and PC88_unstable_20241221.

Thank you.

![image](https://github.com/user-attachments/assets/fd4701c2-820a-4447-a3c9-717419e39d51)

![image](https://github.com/user-attachments/assets/1a86185d-9e53-43ed-832a-89f5cad66dca)

![image](https://github.com/user-attachments/assets/e1b7648f-1493-493a-bdbb-1c61ef625ebc)

![image](https://github.com/user-attachments/assets/7a30969b-50f9-4b88-a7b1-36efc7ebf2f3)

![image](https://github.com/user-attachments/assets/e4047674-6f92-416e-bba1-ec3e1ee065bd)

![image](https://github.com/user-attachments/assets/8377ee3e-c48a-4f7d-baf1-d140cc8d0dba)

![image](https://github.com/user-attachments/assets/0376f363-150c-44fd-8937-6b16a4512052)

**Comment by wwark** [comment]
@saebaryo did you manage to generate user disk ? Thanks

**Comment by saebaryo** [comment]
@wwark 
No, the user disk could not be created.

The message "Can't write to disk" was displayed on the screen.

The disk image I used is a formatted blank 348,848 bytes (340KB) d88 image.

- PC88_unstable_20241221

![image](https://github.com/user-attachments/assets/feb6251b-71c2-4f44-a7d6-9e16f001d9ca)

![image](https://github.com/user-attachments/assets/90bba11e-746a-4dda-a276-3780d43d3354)

![image](https://github.com/user-attachments/assets/7dccfaf4-b6aa-4a42-91ae-d66d187e0295)

![image](https://github.com/user-attachments/assets/90ab4a41-ee50-436b-b343-4d44c428e5e5)

![image](https://github.com/user-attachments/assets/694fe4c9-9aba-4398-86ae-acd224b46aad)

![image](https://github.com/user-attachments/assets/c1c4d8e4-2eea-40ef-9be2-f994d7baa46c)

![image](https://github.com/user-attachments/assets/9b739722-7914-4ee7-9c3c-16b62c6f2c31)

**Comment by wwark** [comment]
same as me. Need to check if I could find a floppy user disk for Xak2 somewhere (maybe the size is different)

**Comment by saebaryo** [comment]
On the M88, the PC-8801 emulator, a user disk can be created.

When a user disk is created, the d88 file size changes.

Formatted Blank Disk (348,848 bytes (340KB)) → Xak 2 User Disk (381,120 bytes(373KB))

[Xak 2 (1990)(Microcabin)(User).zip](https://github.com/user-attachments/files/18221655/Xak.2.1990.Microcabin.User.zip)


I checked that it starts from the starting point with the FDD setup below. (Long waiting time)
- FDD0 : User Disk
- FDD1 : Disk 4

![image](https://github.com/user-attachments/assets/e17e182a-1c93-4dea-bea5-ec9c0f50ef60)
I've confirmed that it's possible to make a save (F5 Key) while playing the game.

However, I confirmed from the attached user disk that it is also not possible to create a user disk in 'MAKE DISK' mode. (It is possible with M88 emulator)
![image](https://github.com/user-attachments/assets/7b039021-0606-4759-8e01-c5f86cf4a98f)
